tests/unit/sentry_testsupport.h

```diff
@@ -7,6 +7,10 @@
 #include <stdio.h>
 #include <string.h>
 
+#ifndef SENTRY_PERFORMANCE_MONITORING
+#    define SENTRY_PERFORMANCE_MONITORING
```

Comment:
cmake should set this flag (which it does). otherwise it very much depends on the include order, which you have seen can break things in surprising non-obvious ways.